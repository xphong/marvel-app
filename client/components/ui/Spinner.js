import React from 'react';

export const Spinner = () => {
  return <div className="loading">Loading&#8230;</div>;
};

export default Spinner;
