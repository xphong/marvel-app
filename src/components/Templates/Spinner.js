import React, { Component } from 'react';

export default class Spinner extends Component{
  constructor(props){
    super(props);
  }

  render() {
    return (
      <div className='loading'>Loading&#8230;</div>
    );
  }
};
