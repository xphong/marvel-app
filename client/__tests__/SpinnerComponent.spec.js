import React from 'react';
import ReactDOM from 'react-dom';

import Spinner from '../components/ui/Spinner';

describe('Spinner Component', () => {
  it('should render Spinner', () => {
    const wrapper = shallow(
      <Spinner />
    );

    expect(wrapper).toMatchSnapshot();
  });
});
