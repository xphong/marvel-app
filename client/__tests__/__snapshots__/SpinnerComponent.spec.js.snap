// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Spinner Component should render Spinner 1`] = `
<div
  className="loading"
>
  Loadingâ€¦
</div>
`;
